demisto_sdk/commands/upload/uploader.py

```diff
@@ -1,41 +1,118 @@
 import demisto_client
 import os
-
-from demisto_sdk.commands.common.tools import print_color, LOG_COLORS, print_v, print_error
+import demisto_sdk.commands.common.constants as constants
+from demisto_sdk.commands.common.tools import print_color, LOG_COLORS, print_v, print_error, get_child_files, \
+    get_child_directories
 from demisto_sdk.commands.unify.unifier import Unifier
 
 
 class Uploader:
-    """Upload the integration specified in self.infile to the remote Demisto instance.
+    """Upload the files specified in self.infile to the remote Demisto instance.
         Attributes:
-            path (str): The path of an integration file or a package directory to upload.
+            path (str): The path of an file or a package directory to upload.
             verbose (bool): Whether to output a detailed response.
-            unify (bool): Whether to unify a package.
             client (DefaultApi): Demisto-SDK client object.
         """
 
     def __init__(self, input: str, insecure: bool = False, verbose: bool = False):
         self.path = input
         self.log_verbose = verbose
-        self.unify = os.path.isdir(self.path)
         self.client = demisto_client.configure(verify_ssl=not insecure)
 
     def upload(self):
         """Upload the integration specified in self.infile to the remote Demisto instance.
         """
+        if os.path.isdir(self.path):
+            list_files_in_dir = get_child_files(self.path)
+            if f'{self.path}/pack_metadata.json' in list_files_in_dir:
+                self.peck_uploader(self.path)
+        else:
+            print_color('Error: Path input is not a valid package directory.', LOG_COLORS.RED)
+        return 0
+
+    def peck_uploader(self, path):
+        list_directories = get_child_directories(path)
+
+        for directory in list_directories:
+
+            if directory.endswith(constants.INTEGRATIONS_DIR):
```

Comment:
Change each if statement to look like this:
`if is_path_of_<entitie_name>_directory(directory):`
For example:
```suggestion
            if is_path_of_integrations_directory(directory):
```
Add to the `tools.py` file all `is_<entitie_name>_dir(directory):` functions. Each will look like this:
```
is_path_of_<entitie_name>_directory(path):
    return os.path.basename(path) == constants.<entitie_name>
```

Reply:
To each individually